test/v1/connectors/AmendOtherReliefsConnectorSpec.scala

```diff
@@ -55,18 +55,27 @@ class AmendOtherReliefsConnectorSpec extends ConnectorSpec {
 
   }
 
-  "doConnector" must {
+  "AmendOtherReliefsConnector" must {
 
-    val request: AmendOtherReliefsRequest = AmendOtherReliefsRequest(Nino(nino), TaxYear.fromMtd(taxYear), body)
+    val outcome = Right(ResponseWrapper(correlationId, ()))
 
     "put a body and return 204 no body" in new IfsTest with Test {
-      val outcome = Right(ResponseWrapper(correlationId, ()))
+      val request: AmendOtherReliefsRequest = AmendOtherReliefsRequest(Nino(nino), TaxYear.fromMtd(taxYear), body)
 
       willPut(url = s"$baseUrl/income-tax/reliefs/other/$nino/$taxYear", body = body)
         .returns(Future.successful(outcome))
 
       await(connector.amend(request)) shouldBe outcome
     }
+
+    "put a body and return 204 no body for a TYS tax year" in new TysIfsTest with Test {
+      val request: AmendOtherReliefsRequest = AmendOtherReliefsRequest(Nino(nino), TaxYear.fromMtd("2023-24"), body)
+
+      willPut(url = s"$baseUrl/income-tax/reliefs/other/23-24/$nino", body = body)
+        .returns(Future.successful(outcome))
+
+      await(connector.amend(request)) shouldBe outcome
+    }
   }
 
```

Comment:
can we update the structure of this spec to match [this](https://github.com/hmrc/individuals-expenses-api/blob/main/test/v1/connectors/CreateAndAmendOtherExpensesConnectorSpec.scala) one? I think it has better word of the tests and is structured in a way that it would be easy to add new pre-tys- and tys-specific tests if needed in the future (which though is probably unlikely, still good practise)